emscripten/npm/src/verovio-logger.js

```diff
@@ -1,15 +1,13 @@
-export const logLevel = {
-    off: 0,
-    debug: 1,
-    info: 2,
-    warning: 3,
-    error: 4,
-};
+export const LOG_OFF = 0;
+export const LOG_ERROR = 1;
+export const LOG_WARNING = 2;
+export const LOG_INFO = 3;
+export const LOG_DEBUG = 4;
 
-export function enableLog(level = logLevel.debug, VerovioModule) {
+export function enableLog(level = LOG_DEBUG, VerovioModule) {
```

Comment:
Do we need a default here? It would be better not to have one. If we need set it to `LOG_WARNING`

Reply:
> I set logLevel.debug as default value for the parameter so you can use enableLog() to print all logs.

We don't need this. But it could be a handy shortcut. Should I remove it to make it consistent with C++ and Python?

I could throw an error instead if no level is passed. Since logging by the emscripten module will be disabled until you set `enableLog` this could be a good idea.

Reply:
Yes, remove it. We can throw an error if you think this is appropriate. (Logging is enabled by default)

Reply:
Okay, changed and pushed it. I will have a look if we need to throw there once this is testable.